2-mandatory/1-writers.js

```diff
@@ -71,7 +73,14 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  writers.forEach(writer => {
+    if (writer.age >= 40
+      && writer.age < 50
+      && writer.alive === false) {
+      console.log(`Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`)
+    }
+  }
+  )
 }
 
```

Comment:
Great use of forEach and of a if statement.  I have used the filter method instead of the if statement. Both solutions work great